src/MSBuildTaskHost/TypeLoader.cs

```diff
@@ -127,10 +127,11 @@ internal static bool IsPartialTypeNameMatch(string typeName1, string typeName2)
         /// any) is unambiguous; otherwise, if there are multiple types with the same name in different namespaces, the first type
         /// found will be returned.
         /// </summary>
-        internal LoadedType Load
```

Comment:
We does this exist? I think it's fine to leave it essentially unchanged, since we don't need to worry about locking an assembly we need unlocked when the task host comes to load it in that task host, but should this whole thing be deleted in favor of the shared version?